files/galaxy/tpv/users.yml

```diff
@@ -1,5 +1,9 @@
 ---
 users:
+  default:
+    scheduling:
+      reject:
+        - ufz-eve-pulsar
```

Comment:
```suggestion
```
If it gives you a better feeling we can leave it, but you are already requiring a tag for your destination which only your two users get.